src/block.tsx

```diff
@@ -19,10 +19,22 @@ import { classNames, getTextContent, getListNumber } from "./utils";
 
 export const createRenderChildText = (
   customDecoratorComponents?: CustomDecoratorComponents
-) => (properties: DecorationType[]) => {
+) => (properties: DecorationType[], blocks: BlockMapType) => {
   return properties?.map(([text, decorations], i) => {
     if (!decorations) {
-      return <React.Fragment key={i}>{text}</React.Fragment>;
+      return (
+        <React.Fragment key={i}>
+          {text.split(/(\n)/).map((t, index) => (
+            <React.Fragment key={index}>
+              {t !== "\n" ? t : <br />}
+            </React.Fragment>
+          ))}
+        </React.Fragment>
+      );
+    }
+
+    if (text === "‣" && decorations[0][0] === "p") {
+      text = blocks[decorations[0][1]]?.value?.properties?.title ?? "";
```

Comment:
I add `blocks` param so we could map page title here, and since we only use `blockMap`, it's not possible to map user mention to username